src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -1992,6 +2003,30 @@ namespace SurfaceGeometry {
 
 	}
 
+	bool
+	isAzimuthDifferent( Real64 azimuth1, Real64 azimuth2 )
+	{
+		return ( std::abs( azimuth1 - azimuth2 ) > 10.0 );
```

Comment:
@mjwitte I noticed that difference comparisons are used in different places also. I am not sure what change you want. Should the function include tolerance as another argument? Do you think the name of the function should change?